packages/SystemsThinkingWorkbench-Test.package/STWResetTest.class/instance/testSimpleReset.st

```diff
@@ -18,8 +18,6 @@ testSimpleReset
 	self assert: (simulation resetState contains: [:element | element = node2]).
 	self assert: (simulation resetState contains: [:element | element = connection1]).
 	
-	simulation stepSimulation.
```

Comment:
"Remove stepSimulation from tests because startSimulation call it."
--> please read merge request descriptions :)